packages/react-composites/tests/browser/chat/hermetic/FileSharing.test.ts

```diff
@@ -2,12 +2,11 @@
 // Licensed under the MIT license.
 import { expect } from '@playwright/test';
 import { sendMessage, waitForMessageDelivered } from '../../common/chatTestHelpers';
-import { dataUiId, isTestProfileStableFlavor, stableScreenshot, waitForSelector } from '../../common/utils';
+import { dataUiId, stableScreenshot, waitForSelector } from '../../common/utils';
 import { buildUrlForChatAppUsingFakeAdapter, DEFAULT_FAKE_CHAT_ADAPTER_ARGS, test, TEST_PARTICIPANTS } from './fixture';
 
+/* @conditional-compile-remove(file-sharing) */
```

Comment:
How does this work? Do we add these test.ts into conditional compilation?

Reply:
Just checked the config file - we do make unittest works for conditional compilation,  but not for e2e test